urbit/umbrel-app.yml

```diff
@@ -35,12 +35,39 @@ torOnly: false
 submitter: ~mopfel-winrux
 submission: https://github.com/getumbrel/umbrel/pull/1246
 releaseNotes: >-
-  Updated Urbit binary to v1.16
-
-  Use new Vere repo, fix binary extraction
-
-  This release is functionally equivalent to v1.15 but is built in the new
-  Vere repo using Bazel instead of Nix with a new and improved release
-  pipeline. It marks the successful extraction of the runtime from the
-  Urbit monorepo. It also adds a new AArch64-specific macOS build and
-  removes the Windows build.
\ No newline at end of file
+  Updated Urbit binary to v1.17
+
+  This release includes a new and improved terminal stack, which is
+  required for compatibility with zuse 416 (urbit-os-v2.130). It remains
+  backwards compatible with the terminal stack as it existed prior to
+  this, but there is one purely visual issue: the prompt may seem to
+  disappear. This should resolve itself once you receive the zuse 416
+  update.
+  
+  Additionally, as of this release, we no longer support Windows
+  platforms. We do not presently have the resources to support non-*nix
+  systems at the level of quality we want to deliver. Notably Windows was
+  already very much a second-class citizen, as apparent from issues like
+  #5822 and others. Windows users may want to run Urbit through WSL
+  instead.
+  
+  Release notes:
+  
+  -   Revamped Urbit’s terminal stack. This release contains the first
+      phase of a longer-running project towards a next-generation terminal
+      experience. Highlights in this release include:
+      
+      -   Support for 2D cursor movement, and by extension 2D screen
+          drawing.
+          
+      -   Minimal mouse-click support.
+      
+      -   Faster pasting of large texts.
+      
+      -   Informal printfs (~&) now get rendered in grey, to visually
+          distinguish them from formal dill output.
+          
+  -   Updated versioning scheme for pre-release binaries.
+  
+  -   The code for the Vere runtime has been moved out of the urbit/urbit
+      repository, and now lives in urbit/vere.

```

Comment:
No problem

Reply:
Could we instead go with something like this, so that user has context of what the update contains but can still follow the link if they want to go deeper?
```
-   Revamped Urbit’s terminal stack. This release contains the first
      phase of a longer-running project towards a next-generation terminal
      experience. Highlights in this release include:
      
      -   Support for 2D cursor movement, and by extension 2D screen
          drawing.
          
      -   Minimal mouse-click support.
      
      -   Faster pasting of large texts.
      
      -   Informal printfs (~&) now get rendered in grey, to visually
          distinguish them from formal dill output.
          
  -   Updated versioning scheme for pre-release binaries.
  
  -   The code for the Vere runtime has been moved out of the urbit/urbit
      repository, and now lives in urbit/vere.

Full release notes here: https://github.com/urbit/vere/releases/tag/vere-v1.17
```

Reply:
This now looks like this:
![Screenshot 2023-01-20 at 17 33 15](https://user-images.githubusercontent.com/1112574/213765666-3aa19471-483c-47eb-be56-1bd474fa501d.png)
So we need an extra empty line under: `-   Updated versioning scheme for pre-release binaries.`

I think we should also drop: `Updated Urbit binary to v1.17` because the version is shown already in the updates UI:
![Screenshot 2023-01-20 at 17 34 31](https://user-images.githubusercontent.com/1112574/213765846-77a07187-9e6d-4c64-82fa-3d7bbe636248.png)

Reply:
Done